{
    "type": "bundle",
    "id": "bundle--3a219c75-95b3-48c7-8e64-c064a4ab0eb9",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--49a4fb1a-7abe-43c3-8572-23433a0d3c7a",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2025-05-21T00:22:55.418927Z",
            "modified": "2025-05-21T00:22:55.418927Z",
            "name": "CVE-2025-48015",
            "description": "Failed login response could be different depending on whether the username was local or central.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2025-48015"
                }
            ]
        }
    ]
}